### ISSUE TITLE
Apple TV Not recognised By Mac

### ISSUE THREAD

**Issue by Bodders07** [NSSpiral/Blackb0x#71]
Anyone how to get it recognised by the mac?

**Comment by TrickShotMLG02** [comment]
have you plugged in the power cable and the usb cable?

**Comment by Bodders07** [comment]
yes, tried both usb ports on mac too nothing happens

**Comment by TrickShotMLG02** [comment]
It might be a problem with your usb cable. I had to try 5 or 6 different cables until i found one which works

**Comment by Bodders07** [comment]
I've tried at least 4 idk why its not showing up, plug a phone in and it works fine

**Comment by Bodders07** [comment]
okay i got a cable what finally worked let me see how it goes

**Comment by Bodders07** [comment]
application freezes every time i plug it in

**Comment by TrickShotMLG02** [comment]
try starting the application, then plugging in the usb cable and after that the power cable
That worked for me some time ago

**Comment by Bodders07** [comment]
same issue, ughh

**Comment by TrickShotMLG02** [comment]
did you try to restore the appletv?

**Comment by Bodders07** [comment]
okay that worked actually took a few goes

**Comment by Bodders07** [comment]
says 'jailbreak is active on device' do  just plug back into tv now?

**Comment by TrickShotMLG02** [comment]
i think so

**Comment by TrickShotMLG02** [comment]
Could you please be so kind and comment, if you successfully installed kodi after the jailbreak?
I am waiting for more than 40min and it doesn't appear.

**Comment by Bodders07** [comment]
i will let you know! its plugged back into tv now and nothings happening as if it isn't plugged in thoigh

**Comment by TrickShotMLG02** [comment]
try to restart the appletv with the remote

**Comment by Bodders07** [comment]
Doesn't seem to be any different yet

**Comment by NSSpiral** [comment]
Can you please post the contents of /var/mobile/Media/blackb0x.log?
You can get it via ssh or from a tool like ifunbox/imazing.

**Comment by NSSpiral** [comment]
> Doesn't seem to be any different yet

Try the new one I uploaded

**Comment by Bodders07** [comment]
ive tried new one just says device already jailbroken

**Comment by Bodders07** [comment]
> /var/mobile/Media/blackb0x.log?
'no such file or directory'

**Comment by NSSpiral** [comment]
Restore before trying the new one, also the blackb0x.log is on device. Can get it by **scp ip-of-apple-tv:/var/mobile/Media/blackb0x.log /path/to/save/blackb0x.log**

**Comment by Bodders07** [comment]
tried restoring first and still nothing after jailbreak

**Comment by Bodders07** [comment]
il just leave it its fine
its not gonna work

**Comment by EdWaQ** [comment]
Got the same problem, looks like a endless loop.

**Comment by EdWaQ** [comment]
I'm waiting now around one hour. The led flashing like hell, the process still running. Apple TV 3

**Comment by EdWaQ** [comment]
Apple TV A1469 *

**Comment by NSSpiral** [comment]
I'm working on fixing a few things at the moment. EdWaQ is your problem kodi not appearing or the Apple TV not showing up in blackb0x?

**Comment by EdWaQ** [comment]
My jailbreak now working, but can't see kodi, I try to ssh my Apple TV and install kodi. I got just some problems with auto install kodi, the app show up my Apple TV.

**Comment by NSSpiral** [comment]
Ok, thanks.

**Comment by EdWaQ** [comment]
How is the standard Password for ssh?

**Comment by NSSpiral** [comment]
Password will be alpine

**Comment by tonystam** [comment]
Thank you very much for this tool! 

I have managed to install Kodi helix on my Apple TV3. These are the steps that I followed:

1. I restored my apple tv 3 to factory settings, then connected it to the TV and let the restoration finish on the TV.
2. I disconnected it from the TV again and connected it to the Mac.
3. I used the Blackb0x tool.
4. When the tool asks to connect it to the TV, I have not removed the power cord. Simply, leaving the device plugged into electricity, I have removed the micro usb cable and plugged the HDMI into the TV.
5. If the device does not restart by itself, press the down key + menu, so it restarts manually.
6. Wait for the main menu to appear. Even though the main menu is displayed, it takes a few minutes for the Kodi icon to appear.

Do you know if there will be a way to update it to version 19.2 / Matrix?

Thanks!

**Comment by NSSpiral** [comment]
Nice, glad it worked for you! 

It is definitely possible to add compatibility back to some newer versions of Kodi but I have a feeling the reason they dropped support is because it simply wasn't worth the performance hit to keep it updated. Will get around to looking at how difficult it is eventually.

**Comment by EdWaQ** [comment]
I'll try later my luck. Thx for the fast answer and the awesome work.

**Comment by dilijasi** [comment]
@tonystam thank you for clear instructions, I followed it to the letter and finally got Kodi working on my atv3! The key step was to un-jailbreak the device and restore it before running the latest version of blackb0x app to jailbreak it again

@NSSpiral thank you for updating the app, I was ready to give up on atv3

**Comment by EdWaQ** [comment]
The led flashing fast, is it normal in the app "to enter DFU mode" ?

**Comment by NSSpiral** [comment]
> The led flashing fast, is it normal in the app "to enter DFU mode" ?

Yep, Menu + Down until flashing, then Menu + Play/Pause until rapidly flashing to enter DFU mode.

**Comment by EdWaQ** [comment]
Awesome it work!!! Finally got my new remote, lost the old one and I was trying everything to dfu with none Apple Remote. No chance for me now everything work very well. Thx for it, next step is to update kodi. Thx a lot for it great work.
